CMakeLists.txt

```diff
@@ -241,6 +279,37 @@ if(NOT NONET)
   target_link_libraries(devilutionx PRIVATE sodium)
 endif()
 
+if(USE_SDL1)
+  target_link_libraries(devilutionx PRIVATE
+    ${SDL_TTF_LIBRARY}
+    ${SDL_MIXER_LIBRARY}
+    ${SDL_MIXER_LIBRARIES}
+    ${SDL_LIBRARY}
+    ${FREETYPE_LIBRARIES}
+    ${PNG_LIBRARIES})
+  if(WARPOS)
+    target_link_libraries(devilutionx PRIVATE
+      -lmikmod
+      -lvorbisfile
+      -lvorbisenc
+      -lvorbis
+      -logg
+      -lFLAC
+      -lbz2)
+  endif()
+else()
+  target_link_libraries(devilutionx PRIVATE
+    SDL2::SDL2main
+    ${SDL2_ttf_LIBRARY}
+    ${SDL2_mixer_LIBRARY}
+    ${FREETYPE_LIBRARIES}
+    ${PNG_LIBRARIES})
+endif()
+
+if(AMIGA AND NOT WARPOS)
+  target_link_libraries(devilutionx PRIVATE -ldebug)

```

Comment:
Shouldn't this only be enabled for when it's compiled in debug mode?

Reply:
the provided SDL libs on http://aminet.net is compiled in debug mode. Don't ask me why. Should be removed

Reply:
Without it I get linking errors:
```
/opt/amiga/lib/gcc/m68k-amigaos/6.5.0b/../../../../m68k-amigaos/bin/ld: /opt/m68k-amigaos/lib/libSDL.a(SDL_video.go-00000010.o): in function `SDL_SetVideoMode':
�$��:32534: undefined reference to `kprintf'
/opt/amiga/lib/gcc/m68k-amigaos/6.5.0b/../../../../m68k-amigaos/bin/ld: �$��:32534: undefined reference to `kprintf'
/opt/amiga/lib/gcc/m68k-amigaos/6.5.0b/../../../../m68k-amigaos/bin/ld: �$��:32534: undefined reference to `kprintf'
/opt/amiga/lib/gcc/m68k-amigaos/6.5.0b/../../../../m68k-amigaos/bin/ld: �$��:32534: undefined reference to `kprintf'
/opt/amiga/lib/gcc/m68k-amigaos/6.5.0b/../../../../m68k-amigaos/bin/ld: �$��:32534: undefined reference to `kprintf'
/opt/amiga/lib/gcc/m68k-amigaos/6.5.0b/../../../../m68k-amigaos/bin/ld: /opt/m68k-amigaos/lib/libSDL.a(SDL_video.go-00000010.o):�$��:32534: more undefined references to `kprintf' follow
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/devilutionx.dir/build.make:516: devilutionx] Error 1
make[1]: *** [CMakeFiles/Makefile2:77: CMakeFiles/devilutionx.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
```

Reply:
Yeah, someone has been a bad boy and not wrapped debug code with ifdef's in the Amiga libSDL12 repo. ;)